ansible/configs/kni-osp/env_vars.yml

```diff
@@ -0,0 +1,70 @@
NFRFbTZsakc3QzZwLTdsRnU5SEgyb1N2ajZjaEdMMFhWUGxUVDF2dExIVXRIcUdUeUpVdXk4alo4ZGt0czhzQmNOemhDZHhLRndYejRoR1RkUmF5ZTVZaU11LUZBZFRiNU9nd0JBbG9fa2twOUlnS213YmpIbXgwY1dDZ3lJcTcweDNkUE5KSnZTcnotbjF1bDJhT0FoNTc5U1IzU0Z3OTdZNzlZamxYc3BsTlJwRVhKVEprT29jVGF0QklKSzRHd2F5elcwc0gyZjY1OG1hYXlqWXBzbzRKcGd0VEJEd0JXNUF5Um5TVmpzcnlBb1NWbGZWTGp5cndaRkI3SGh0X3FaYThiTnNJ", "email": "bschmaus@redhat.com" }, "registry.redhat.io": { "auth": "NzY2MjIyNXx1aGMtMUpqalU3YXdsQXE4WThNWlJZeExNd1JKZER2OmV5SmhiR2NpT2lKU1V6VXhNaUo5LmV5SnpkV0lpT2lKa056Vm1NelpqTkRjMlpqZzBOV0kxWWpBMlpUVmhaVGd3Tm1SaVpUUTROU0o5LkUyVld5TkRubWFNRnZIbGM0Z0JHZTlMWUk5TC0zdlZVUmVuX2JXbkktZWRpdTBFbG5fLWM0QnlIWlZJcm0zLUlmeTl2b3ZWbEM4TE5qV1lMam9jQWI4Y0VNb3dSZnBZdEN0R0IxT0ZweS10dzZqV2lnSHlmSHhvem53cUZyNDVUTXZaQ041QlljNEg2X0xTYkUzejNIaVUxRDBuZzZzRmZiS2RyTUxKcklFYlR6QTFVQnUwTzRqc3ExSThBLUliV1YxaHlRclJLZ2lsUmdxQXFCdnVPdS0yWllERjYzUWtkWmxDTElPUTR5RUg1dUpUb3JYazhhWmxtRTdRelVCUjBTQkJ4eEdzWDBUclhqR0pWaWMxdzFJQ0JmVjJ3Vjd2NWlnYkROYTNYLVRLU3NMNjRmN19mcWtuVE83ZWsxbW16UlNtb2R1U203dHF5T1BuOXpiV19PQkx4ZTR4Wm00MlppNDc3NUhXU0pLaW5zZ2xvVW1FSU9HZm5XYVVPMFU4aWhWT0ZXUVNnZDZBa2JWaTJmNDN6NWJYeWZva3Rzb1BxZFpUNFRFbTZsakc3QzZwLTdsRnU5SEgyb1N2ajZjaEdMMFhWUGxUVDF2dExIVXRIcUdUeUpVdXk4alo4ZGt0czhzQmNOemhDZHhLRndYejRoR1RkUmF5ZTVZaU11LUZBZFRiNU9nd0JBbG9fa2twOUlnS213YmpIbXgwY1dDZ3lJcTcweDNkUE5KSnZTcnotbjF1bDJhT0FoNTc5U1IzU0Z3OTdZNzlZamxYc3BsTlJwRVhKVEprT29jVGF0QklKSzRHd2F5elcwc0gyZjY1OG1hYXlqWXBzbzRKcGd0VEJEd0JXNUF5Um5TVmpzcnlBb1NWbGZWTGp5cndaRkI3SGh0X3FaYThiTnNJ", "email": "bschmaus@redhat.com" }, "registry.svc.ci.openshift.org": { "auth": "c3lzdGVtLXNlcnZpY2VhY2NvdW50LWtuaS1kZWZhdWx0OmV5SmhiR2NpT2lKU1V6STFOaUlzSW10cFpDSTZJaUo5LmV5SnBjM01pT2lKcmRXSmxjbTVsZEdWekwzTmxjblpwWTJWaFkyTnZkVzUwSWl3aWEzVmlaWEp1WlhSbGN5NXBieTl6WlhKMmFXTmxZV05qYjNWdWRDOXVZVzFsYzNCaFkyVWlPaUpyYm1raUxDSnJkV0psY201bGRHVnpMbWx2TDNObGNuWnBZMlZoWTJOdmRXNTBMM05sWTNKbGRDNXVZVzFsSWpvaVpHVm1ZWFZzZEMxMGIydGxiaTAxZEdkbU55SXNJbXQxWW1WeWJtVjBaWE11YVc4dmMyVnlkbWxqWldGalkyOTFiblF2YzJWeWRtbGpaUzFoWTJOdmRXNTBMbTVoYldVaU9pSmtaV1poZFd4MElpd2lhM1ZpWlhKdVpYUmxjeTVwYnk5elpYSjJhV05sWVdOamIzVnVkQzl6WlhKMmFXTmxMV0ZqWTI5MWJuUXVkV2xrSWpvaVlqZzNNRGt4WmpZdE5qRXlNeTB4TVdVNUxXRTJNVGt0TkRJd01UQmhPR1V3TURBeUlpd2ljM1ZpSWpvaWMzbHpkR1Z0T25ObGNuWnBZMlZoWTJOdmRXNTBPbXR1YVRwa1pXWmhkV3gwSW4wLm51VGR0RlczRENHcFpvT0pCbU45VjQwWG1wbmlZRE9tUnI2Z05vNGVwRVBrb1lDXzk1YmhWX0ttYjhoTnprOTNVTGtDNnJXNTVjTXFQMVM4RHh3QWw0RUxRZ2NFZXIyalBJLXZBNGUzdlZ5cHNLbS1XSkFxcWo2OGhNN0Z4ekMzRGgxY19lN19EQkJLOWtxZmcyRzZiNTJXQmI2RUhsODg2Q2Nza3JBVm1fbmprNS14ay1Ma1hSM3lXNW5JeXlZdXhNVGg1LUNMd3lQQy1yLVIzeklzdnlWelNPVTgyeUJaaE1tUmc3enUtOWlydThENHdqRFJQclhiSm1FV3lBM1FIUlJ2VTJuci01MTFEeEhEbWhtNW14YU0tSFA4emk3SU8zVEU5SU55S3BqTmo5eTIwNmtFN0NNSVNMWmRWWFl3MkpIQ1BmSmJQMHNJY3V0dnFvOTdGdw==" }, "quay.io/openshift-cnv": {"username": "openshift-cnv+beta", "password": "HEEF8PBP6PKN33YINWP4ATD4P4VXKQFIWUKPU0XOCY22YI3LIHIA2H8YK0LQ0E7U", "auth": "b3BlbnNoaWZ0LWNuditiZXRhOkhFRUY4UEJQNlBLTjMzWUlOV1A0QVRENFA0VlhLUUZJV1VLUFUwWE9DWTIyWUkzTElISUEySDhZSzBMUTBFN1U="} } }'
```

Comment:
Is this a legit pull secret? This doesn't belong in a public repo.

Reply:
I was told by gpte team members they would replace it with a generic one while getting it into a catalog item ? (when asking how I can get the user to provide his own)

Reply:
Please make sure you contact the owner of that secret to revoke it ASAP

Reply:
@stencell can you resolve if this is ok now?